src/map/navi.cpp

```diff
@@ -75,39 +72,13 @@ static enum directions walk_choices [3][3] =
 /// @name A* pathfinding related functions
 /// @{
 
-/// Pushes path_node to the binary node_heap.
-/// Ensures there is enough space in array to store new element.
-
-static void heap_push_node(struct node_heap *heap, struct path_node *node)
```

Comment:
we could keep these functions and just put the repeated lines here:
```
static void heap_push_node(std::vector<path_node *>& heap, path_node *node) {
    heap.push_back(node);
	std::push_heap(heap.begin(), heap.end(), min_heap_comp);
}
```
then we just call heap_push_node(heap, &tp[i]);